addons/udes_security/models/blocked_file_type.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class BlockedFileType(models.Model):
+    """
+    Blocked file type which the user is prevented from uploading or downloading
+    """
+
+    _name = "udes.blocked_file_type"
+    _description = "Blocked File Type"
+    _order = "name"
+
+    _sql_constraints = [("name_uniq", "unique (name)", "The File Type must be unique")]
+
+    name = fields.Char("File Type", required=True, index=True)
+    description = fields.Char("Description")
+    active = fields.Boolean("Active?", default=True)
+
+    def _format_name(self, name):
+        """Convert supplied name to lowercase if needed"""
+
+        formatted_name = name or ""
+        if formatted_name:
```

Comment:
is there anypoint in the ifs why not just do the lower and replace if it doesn't change it doesn't matter

Reply:
I suppose not really, will go change this.

Reply:
Have now simplified this and removed the unnecessary if statements. It now uses one inline if which is used just incase ```name``` is ```False```.